packages/perseus/src/widgets/radio/__stories__/answerlessRadio.stories.tsx

```diff
@@ -0,0 +1,90 @@
+// NOTE(Tamara): ServerItemRenderWithDebugUI does not work with the same props
```

Comment:
Rather than make a new file, we need to update `radio.stories.tsx` to use `ServerItemRendererWithDebugUI`. The problem isn't that we need stories for both `ServerItemRendererWithDebugUI` and `RendererWithDebugUI`, the problem is that `RendererWithDebugUI` needs to be deprecated and replaced with `ServerItemRendererWithDebugUI`

Reply:
I wanted to preserve Controls functionality for the answerful stories and wasn't sure how to do that with ServerItemRendererWithDebugUI. So I changed it from converting all stories to ServerItem to just the answerless stories in the new file. Just took a second attempt and was able to preserve the Controls functionality for all stories. The answerless ones just need to turn `startAnswerless` off or click the Check button first to get full data.